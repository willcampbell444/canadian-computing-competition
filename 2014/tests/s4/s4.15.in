100
6543210
0 0 1000000000 1000000000 623953
881894098 61956686 923919647 656234251 512406
608013403 4478624 719806026 947050373 284502
884515902 176943573 940707665 699099635 758895
49189760 98559064 776820240 464666760 787128
24482147 302051245 498709586 976316240 911786
623503386 57998935 753110825 648677982 346234
26335824 685704207 241847134 830536649 732822
190905917 8156979 985270174 982390980 371224
255027639 19636100 897425026 244742294 529832
345767951 231186869 759773054 567936710 940896
88816360 28406433 89137255 126721403 528376
93119188 331592632 357285253 528953353 172132
46538256 70395046 452022588 139472009 379580
865794726 163996807 945335160 855746389 559158
191177024 129283408 608927433 476689650 254333
451834209 268353449 859836853 860510902 737222
523450470 49747225 569547473 184526110 927696
499343210 95221188 507172895 692234516 577807
103368822 424993061 345860073 591494894 749499
76343900 64173234 190258332 68618735 731623
351121455 13215423 862257796 747110610 781914
93896692 425745786 814451510 868688739 298999
430683870 129407632 979472611 534002015 603454
64534130 145864369 182376648 538137245 591750
344605127 673780740 720740261 807952109 979176
455113142 160711876 980042084 897293820 50585
112773624 87148012 805056740 305705107 828304
242687513 142668796 593467676 570563916 230936
391804643 517322174 574686220 668751777 470294
435679632 341317333 591577788 946064146 158013
632207750 108517037 750673824 914035734 32926
884076944 34822793 944108545 668626711 673867
131385124 315009266 608555164 369568530 527001
38732489 375263327 67392197 875371545 69797
95798776 63266945 116117928 324359352 116172
272202986 775400786 275804782 974239426 223500
393794378 104977831 716821747 239825353 369293
715944554 529228131 736020537 791341833 398684
378900043 467429113 381570811 676987924 655441
238306388 355491452 570715623 951440040 499912
603244966 303560864 868875610 467829269 762817
43356459 488028658 850585691 788049137 177704
404930009 282967114 639033585 671727642 473734
346340542 153560362 519232113 264789217 127281
499088360 106924532 784263625 657098668 383396
534729749 127018583 657715557 131295587 557942
297220610 98279517 660281627 132807926 891640
511461220 264995187 595519325 544664178 490872
38609570 370964790 363709954 927716094 181483
769647572 745299175 888638460 965715301 99853
274003505 971952561 846056838 981028057 90912
475545493 351945296 862586137 616082241 609228
107942624 50468050 399502520 153973816 109456
28526353 93011790 712468724 139535552 853904
370867713 873236219 634121920 969087375 511356
79050515 4144824 671192703 895602076 563458
322311653 683665130 824641527 931172815 660948
304192535 198350817 787544369 316476515 772565
598874702 31246219 987547259 598652929 176059
16769391 329864138 866636757 522779974 406335
252814593 9563165 717182591 125189362 607038
417480312 762512229 508094347 939688135 723486
17393138 268677568 73273572 439285190 242576
179873439 244708638 904533139 268892876 956838
53594498 372729593 271365206 967062987 454446
165167564 67871156 230177587 976639656 685529
304428808 21907479 417786233 609159181 788397
399041808 897216054 942764437 907528287 517185
155234008 730057941 210800893 972808191 240105
132738840 695557847 878749784 853641487 204981
183969382 40191221 451808317 389930173 793808
762709685 116889020 779691148 531770046 200714
627470164 463263181 787808649 746322771 557624
9397502 177493076 644402976 720769005 321228
388021970 157626904 655566326 370392430 945198
837461621 336538674 911170491 402620287 34561
441082433 207376878 650969452 951880566 212659
134433208 64558083 530334308 852104175 405961
267495733 533936422 428898023 910489938 262320
219306291 798877779 671053803 964619817 191695
529296301 5648226 580705419 750268358 432493
124170631 374668912 640248554 764358801 334174
801628846 412391190 983043024 456592023 980040
269497300 398092771 333624985 948299157 770506
4929798 250719593 27079913 658543206 262716
705862413 95360783 826523121 729892457 916329
503220712 823783561 525324512 839887892 772307
10109094 45313323 17303555 910539734 824010
63625363 224249205 421617618 697598314 571144
143894109 86757769 616382929 947677404 897808
586097077 819062929 620453627 930985997 778116
106266230 221777686 572053673 896107270 424251
37078222 196398102 310122104 279373853 513454
162698079 390930843 677025008 719914316 180707
177528905 409800836 370834436 666499230 573302
481391463 496982168 895906756 968398970 372972
576560052 216122303 804425548 798561894 682638
231559095 366319879 695568983 674389853 549084
82489751 716783059 617014958 764846322 79784
