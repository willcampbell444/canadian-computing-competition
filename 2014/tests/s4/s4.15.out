424801369581089064
